gustaf/io/mfem.py

```
@@ -107,97 +107,176 @@ def extract_values(fname, start_index, n_lines, total_lines, dtype):
 
-        Parameters
-        ------------
-        array: NumPy array
 
-        Returns
-        ------------
-        str
-        """
-        row_strings = [" ".join(map(str, row)) for row in array]
-        return "\n".join(row_strings)
+def _export_2d(mesh):
```

Ruff (0.14.3)

Missing return type annotation for private function `_export_2d`

(ANN202)